### PR TITLE
Beta Power Bomb Room: in-room Speed Kill

### PR DESCRIPTION
**Pull request by dragonsinth**

### CODE REVIEW

File: region/brinstar/red/Beta Power Bomb Room.json [hunk 1]
```diff
@@ -305,6 +305,28 @@
...
+              "openEnd": 0
+            }
+          }
+        ]}
+      ],
+      "clearsObstacles": ["A"],
+      "note": [
+        "Manipulate the Sidehoppers to a wall and synchronize their jumps.",
+        "Roll away just as they jump towards Samus to the opposite wall.",
+        "Shinecharge to the right, and jump into the Sidehoopers when Samus acquires blue speed."
```
Comment (dragonsinth) [inline region/brinstar/red/Beta Power Bomb Room.json:327]:
```suggestion
        "Shinecharge towards the Sidehoopers, jumping into them when Samus acquires blue speed."
```

**Review by kjbranch — COMMENTED** [review]
I dont think this is useful. If you have morph and the skill to get a 16 tile shortcharge, you can just kill the hoppers.

**Comment by dragonsinth** [comment]
Fair. It's probably more than 16 tho, my shortcharges are ok but not wonderful (I don't stutter).

File: region/brinstar/red/Beta Power Bomb Room.json [hunk 2]
```diff
@@ -305,6 +305,28 @@
...
+          "Morph",
+          "canCameraManip",
+          {
+            "canShineCharge": {
+              "usedTiles": 16,
+              "openEnd": 0
+            }
+          }
+        ]}
+      ],
```
Comment (osse101) [inline region/brinstar/red/Beta Power Bomb Room.json:322]:
```suggestion
        "Morph",
        "canCameraManip",
        {
          "canShineCharge": {
            "usedTiles": 16,
            "openEnd": 0
          }
        }
      ],
```

**Review by osse101 — COMMENTED** [review]
I don't see how you synchronize the hoppers.  I can do the strat but it doesn't feel safe.  I'd rather keep 2 against the wall and kill whichever one follows me with power beam and then speed kill the remaining.  But that is most of the way towards killing all of them with beam shots.

File: region/brinstar/red/Beta Power Bomb Room.json (hunk 2 above)
Comment (dragonsinth) [reply region/brinstar/red/Beta Power Bomb Room.json:322]: oops yeah

**Comment by dragonsinth** [comment]
Maybe we should scrap this in favor of a comeInShineCharging based strat?  What would that look like?

**Comment by dragonsinth** [comment]
Gonna close this for now.

**Pull request closed without merging**